hogben/tests/test_fisher.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pytest
+
+from hogben.simulate import simulate
+from hogben.utils import fisher
+from refnx.reflect import SLD
+from unittest.mock import patch
+
+
+@pytest.fixture
+def model_fixture():
+    """Define a bilayer sample, and return the associated refnx model"""
+    # Define sample
+    air = SLD(0, name='Air')
+    layer1 = SLD(4, name='Layer 1')(thick=60, rough=8)
+    layer2 = SLD(8, name='Layer 2')(thick=150, rough=2)
+    substrate = SLD(2.047, name='Substrate')(thick=0, rough=2)
+    layer1.thick.bounds = (40, 70)
+    layer2.thick.bounds = (100, 180)
+    layer1.rough.bounds = (6, 10)
+    layer2.rough.bounds = (1, 3)
+    sample = air | layer1 | layer2 | substrate
+
+    # Define model
+    angle_times = [(0.7, 100, 5), (2.0, 100, 20)]  # (Angle, Points, Time)
+    model, _ = simulate(sample, angle_times)
```

Comment:
this would probably be better just calling refnx directly:
` model = refnx.reflect.ReflectModel(sample, scale=scale, bkg=bkg, dq=dq)`

Reply:
I tried to do this for the refl1d fixture as well, but refl1d is a bit less convenient to work with and requires some processing which is all handled in `simulate`. 

I could move that logic to the refl1d fixture as well, so that a call to simulate with a blank `_` parameter is not needed, but I still used the `simulate` method for the refl1d fixture to keep it cleaner. I'm happy to just repeat that logic in the fixture though if that's preferred.